backend/tests/v5/helper/services.js

```diff
@@ -592,6 +592,12 @@ ServiceHelper.generateTicket = (template, internalType = false, container) => {
 	return ticket;
 };
 
+ServiceHelper.generateImportedComment = (author = ServiceHelper.generateRandomString()) => ({
+	...ServiceHelper.generateComment(author),
+	originalAuthor: ServiceHelper.generateRandomString(),
```

Comment:
agreed that we can remove `createdAt`

But what's the benefit of merging the 2 functions? I think it's clearer this way and it's not like we're duplicating the 2?

Reply:
I think it's more annoying to call as well if say we put isImport as a param after author then we'd have to do `generateComment(false, true)`  instead of just calling `generateImportedComment()` everywhere.